int32 NUM_MEASUREMENTS
float64[] angles
float64[] measurements
